perfkitbenchmarker/benchmark_spec.py

```diff
@@ -229,7 +229,19 @@ def ConstructVirtualMachines(self):
 
         if DISK_SPEC in group_spec:
           disk_spec_class = disk.GetDiskSpecClass(cloud)
-          disk_spec = disk_spec_class(**group_spec[DISK_SPEC][cloud])
+          disk_spec_dict = group_spec[DISK_SPEC][cloud]
+          # We have to translate the data_disk_type flag here because
+          # this is where we know for sure what cloud we're using, and
+          # therefore we're able to pick the right translation table.
+          if FLAGS['data_disk_type'].present:
```

Comment:
I am seeing a bunch of references to the literal string 'data_disk_type' here, I think it'd be more robust and useful if we can define a constant representing the literal string and use the constant here. (and preferrably the constant should be defined right at the place where the flag is defined).

Or is it possible to do FLAGS.data_disk_type.GetFlagName() sort of thing here?